frappe/utils/change_log.py

```diff
@@ -182,32 +183,34 @@ def parse_latest_non_beta_release(response):
 
 	return None
 
+
 def check_release_on_github(app):
-	# Check if repo remote is on github
 	from subprocess import CalledProcessError
+
 	try:
+		# Check if repo remote is on github
 		remote_url = subprocess.check_output("cd ../apps/{} && git ls-remote --get-url".format(app), shell=True).decode()
 	except CalledProcessError:
 		# Passing this since some apps may not have git initializaed in them
-		return None
+		return
 
 	if isinstance(remote_url, bytes):
 		remote_url = remote_url.decode()
 
 	if "github.com" not in remote_url:
-		return None
+		return
 
 	# Get latest version from github
 	if 'https' not in remote_url:
-		return None
+		return
```

Comment:
Do these checks support git SSH urls? If not, can you extend this to something similar to `is_git_url` from [bench](https://github.com/frappe/bench/blob/develop/bench/app.py#L61)?

Everything else looks good to me 👍

Reply:
Nada on the SSH URLs.

Do you want me to port the function into Frappe? Or import from bench? Because that is not currently an app dependency.

Reply:
Just porting the function to `frappe.utils` should be enough. No need for another dependency. 👍

Reply:
I've had it return for git URLs explicitly for now, cause I was trying to find an elegant way to retrieve the organization name, but couldn't find anything clean.

Reply:
There's a non-standard module [parse](https://github.com/r1chardj0n3s/parse) that does that.

![Screenshot 2020-11-24 at 1 03 12 PM](https://user-images.githubusercontent.com/36654812/100062414-8564b680-2e55-11eb-9c79-59e93410a6d3.png)

Reply:
If we're okay with using a dependency, I would suggest using [giturlparse](https://pypi.org/project/git-url-parse/) and just parse the remotes directly.

![image](https://user-images.githubusercontent.com/13396535/100217976-190fb300-2f3a-11eb-83ad-b7ad924aa9e6.png)

Reply:
If that works better, sure!

Reply:
@gavindsouza, I've added the new library, have a look